draft-ietf-scitt-architecture.md

```diff
@@ -142,7 +142,7 @@ The rise in popularity of verifiable data structures as a mechanism to make acto
 
 This document defines a generic, interoperable and scalable architecture to enable transparency across any supply chain with minimum adoption barriers.
 It provides flexibility, enabling interoperability across different implementations of Transparency Services with various auditing and compliance requirements.
-Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Consumers will be able to verify them.
+Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Consumers[TODO: Fix Resolve Consumer] will be able to verify them.
```

Comment:
I would recommend this or I can drive alignment after these fixes are merged and continue the work in #120.

```suggestion
Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Verifiers and Relying Parties will be able to verify them.
```

Reply:
Thanks, @aj-stein-nist 
I reduced this PR to be more about consistency, and suggest we do a separate PR to resolve using Consumer or Relying Parties. I do think we can do something better, but would like to track that one separately.